packages/manager/modules/exchange/src/account/add/account-add.html

```diff
@@ -17,363 +14,898 @@
+                                            </select>
+                                            <span
+                                                class="oui-icon oui-icon-chevron-down"
+                                                aria-hidden="true"
+                                            ></span>
+                                        </label>-->
+                                        <input
+                                            class="w-100 oui-input"
+                                            id="contactInfoMobile"
+                                            name="contactInfoMobile"
+                                            type="text"
+                                            data-ng-pattern="$ctrl.ACCOUNT_WORLD_PHONE_REGEX"
+                                            data-ng-model="$ctrl.telephonyModel.mobile.number"
+                                            data-ng-change="$ctrl.onCountryPhoneCodeChange('mobile')"
+                                            data-ng-required="$ctrl.telephonyModel.mobile.selected"
+                                            data-oui-tooltip="{{:: 'exchange_account_add_phone_help_text' | translate}}"
+                                        />
+                                    </div>
+                                    <div class="oui-field__footer">
+                                        <div
+                                            class="oui-field__error"
+                                            data-ng-if="$ctrl.newAccountForm.contactInfoMobile.$error.required"
+                                            role="alert"
+                                        >
+                                            <span
+                                                class="oui-icon oui-icon-error"
+                                                aria-hidden="true"
+                                            ></span>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+
+                            <!--Fax field-->
+                            <div>
+                                <label
+                                    class="control-label"
+                                    for="contactInfoFax"
+                                    data-ng-bind=":: 'exchange_account_add_contact_fax_label' | translate"
+                                ></label>
+                                <div class="oui-field">
+                                    <div
+                                        class="w-100 oui-input-group_inline"
+                                        data-ng-class="{ 'oui-field_error has-error': $ctrl.newAccountForm.contactInfoFax.$invalid && $ctrl.newAccountForm.contactInfoFax.$touched }"
+                                    >
+                                        <!-- TODO: can be used later once API are completed
```

Comment:
Stil needed ?

Reply:
Yep still needed. API are not yet completed.